itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java

```diff
@@ -235,10 +235,7 @@ public WarehouseInstance run(String command) throws Throwable {
       driver.run(command);
       return this;
     } catch (CommandProcessorException e) {
-      if (e.getCause() != null) {
-        throw  e.getCause();
-      }
-      throw e;
+      throw e.getCause();
```

Comment:
we need not to revert this change right? this is just a null check..

Reply:
I would keep the null check, so next time we might have a little bit more info when there is an error

Reply:
Reverted.